Formula/gambit.rb

```diff
@@ -0,0 +1,39 @@
+class Gambit < Formula
+  desc "Software tools for game theory"
+  homepage "http://www.gambit-project.org"
+  url "https://github.com/gambitproject/gambit/archive/v15.1.1.tar.gz"
+  sha256 "fb4dce2f386e46bbfc72cb75471f43716535937c96ad5a730cad22f97c6a65e6"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "wxmac"
+
+  def install
+    system "aclocal"
+    system "glibtoolize"
+    system "automake", "--add-missing"
+    system "autoconf"
+    system "./configure", "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+
+    share.mkpath
+    share.install "contrib"
+  end
+
+  test do
+    system bin/"gambit-enumpure", share/"contrib/games/e02.efg"
```

Comment:
What are these tests doing, out of curiosity?

Reply:
Each test invokes one of Gambit's command-line tools for the given strategic game file, and produces the results on stdout. The intent is really just to make sure the tools can at least run with real (albeit trivial) input and not segfault.

More specifically, `gambit-enumpure` (for example) searches for pure-strategy Nash equilibria for the supplied strategic game. The game referenced by the first test (`e02.cfg`) looks like this:

```
EFG 2 R "Selten (IJGT, 75), Figure 2" { "Player 1" "Player 2" }
""

p "" 1 1 "(1,1)" { "R" "L" } 0
t "" 1 "Outcome 1" { 1, 1 }
p "" 2 1 "(2,1)" { "R" "L" } 0
t "" 2 "Outcome 2" { 0, 2 }
p "" 1 2 "(1,2)" { "r" "l" } 0
t "" 3 "Outcome 3" { 0, 3 }
t "" 4 "Outcome 4" { 2, 0 }
```

This produces the following output, which includes a banner by default:

```
Search for Nash equilibria in pure strategies
Gambit version 15.1.1, Copyright (C) 1994-2014, The Gambit Project
This is free software, distributed under the GNU GPL

NE,1,0,0,0,1,0
```